pkg/queue/manager.go

```diff
@@ -108,6 +108,11 @@ func (m *Manager) UpdateClusterQueue(cq *kueue.ClusterQueue) error {
 	}
 	// TODO(#8): recreate heap based on a change of queueing policy.
 	cqImpl.update(cq)
+
+	// TODO: Add more fine-grained judgments
```

Comment:
```suggestion
	// TODO(#8): Selectively move workloads based on the exact event.
```

Reply:
The rest of the cohort could be retried too.

Reply:
Thanks updated.